### ISSUE TITLE
Graph. Filter out by Tag

### ISSUE THREAD

**Issue by TheOneValen** [foambubble/foam#744]
I would find it very helpful if the graph view would allow me to filter (in/out) nodes based on tags. Other properties (folder etc.) would also be quite cool.

**Comment by Tenormis** [comment]
This feature is very useful. Doesn't anyone else need it? Hold it up

**Comment by Tenormis** [comment]
@riccardoferretti show your talents !!

**Comment by micharris42** [comment]
At the moment, it is possible to filter the graph view, but this is done with the Type note property (as explained [here](https://foambubble.github.io/foam/user/features/graph-visualization#filter-view)). However, this appears to be the _only_ Foam function that makes use of the Type note property.

I believe it is counterintuitive to have Tags be one of the chief organizational features of Foam ... right up until a user opens the Graph View, and is then required to interact with the entirely separate organizational feature of Types. It is especially egregious considering Types do not appear to serve any purpose towards organization outside of the Graph View.

@riccardoferretti, this might be beyond the scope of the original feature request, but possibly there needs to be a discussion as to whether the note Tags and note Types features may be combined. At the moment, users rely on Tags while in the main Foam interface, but have to switch over to Types when interacting with the Graph View.

Combining these two features into one would allow users have a single organizational tool for their notes that can be used throughout Foam.

Thank you.

**Comment by C0DECHEF** [comment]
@slimhk45 @micharris42 @TheOneValen Here is a related [discussion](https://github.com/foambubble/foam/discussions/1442#discussion-8119317)

**Comment by slimhk45** [comment]
@micharris42 Even simpler than that. Foam should aim to be Obsidian-compatible. Obsidian allows graph to be filtered by any property defined in the frontmatters. This is the best and most intuitive design.

**Comment by riccardoferretti** [comment]
Yes, there are a few features that would make the graph more helpful. For now I am collecting those (see `foam-dataviz` tag for an overview), also happy for some support building some of these

**Comment by riccardoferretti** [comment]
This is now possible in Foam, either by passing a filter to the `show-graph` command, or by creating groups inside the graph view itself.
